guide/src/meta-variables.md

```diff
@@ -1,8 +1,10 @@
 # Meta variables
 
+If you consider a macro invocation like a function call, then meta variables are like parameters.
+
 [[_TOC_]]
 
-If you consider a macro invocation like a function call, then meta variables are like parameters.
+## Meta variables
```

Comment:
I guess I am already on the record that I think meta variables definitely overdo it and should be removed entirely as this kind of abstraction is what the programming languages themselves should be used for?

Reply:
Yes, you are 😉 

Maybe the fact that I never used them so far should convince me to really remove them now... (although not used Yarner in a ton of projects...)

Reply:
Metavariables are very useful to literate configurations: usually configuration files don't allow loops and having the metavariables feature could be very helpful to avoid rewriting the same thing many times changing only one piece.